docs/user-guide/configuring.md

```diff
@@ -23,7 +23,7 @@ Please note that supporting JSX syntax is not the same as supporting React. Reac
 By the same token, supporting ES6 syntax is not the same as supporting new ES6 globals (e.g., new types such as
 `Set`).
 For ES6 syntax, use `{ "parserOptions": { "ecmaVersion": 6 } }`; for new ES6 global variables, use `{ "env":
-{ "es6": true } }` (this setting enables ES6 syntax automatically).
+{ "es6": true } }` (this setting enables ES6 syntax automatically, but not vice-versa).
```

Comment:
I'm not sure the meaning of "vice-versa" is clear here. It might be better to be explicit, e.g.:

> `{ env: { es6: true } }` enables ES6 syntax automatically, but `{ parserOptions: { ecmaVersion: 6 } }` does not enable ES6 globals automatically.

Reply:
I agree your wording is clearer. I've amended the commit. I'm glad to be involved and keen to see this merged.

Reply:
@maxwell-k did you forget to push the commit? I'm seeing you have resolved the conversion.

Reply:
Oh dear, how embarassing I pushed an amended commit just not one with the new wording. Should be fixed now, let's try that again! 

Thanks for your patience.